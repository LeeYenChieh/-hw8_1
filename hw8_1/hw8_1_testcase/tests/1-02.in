NYZcomXmCPzJWFEb40taCea 2USc5uhOHOdt2tLOOzrsLEF5RC2ASrPLcXTkQoje1ZNn8vSx9pFH40
TNZmPQoZj7jVpOgqXQMKdilq5krM3787dR3nB9hYbzI1NckQP 57
NYZcomXmCPzJWFEb40taCea display
NYZcomXmCPzJWFEb40taCea display
C1YxeumCyEwWWWzrV6gD7Q1Tvf CkpBceIHQCal
mTHIlufGcEss87uB2q7dm0Rnh8 R3Ltj72rLTocdv5dC2r8gs360r73VJZyXmNQfN8vlT2A7qqcR1
mTHIlufGcEss87uB2q7dm0Rnh8 7aJbTR6eTygopE8oJ4blvdxro14QmDY1GBoKsCCf2Z39ngVpBwT4sUHi6u
b85nQisGOUZdikWveMy8 A1z8lCNbFBJSTJ85UIJHq5Ss1UjXSJklb0IsNjhXlw4gmx5pN
xeG6Rqhsz6yczTPtxmXOoaMv1CnuFCN1pRnWuoxIxIqfkx5ZFaDOVmtoccP8M ghJHc76pho7D5KeCnYlEl6SMdxESzT78OL0TD8f8knQZJml2
b85nQisGOUZdikWveMy8 u6E30stfipn37MhFYw7cC8GQ7F7YT4cC
