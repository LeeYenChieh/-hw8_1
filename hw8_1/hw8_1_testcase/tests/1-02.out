2USc5uhOHOdt2tLOOzrsLEF5RC2ASrPLcXTkQoje1ZNn8vSx9pFH40 
2USc5uhOHOdt2tLOOzrsLEF5RC2ASrPLcXTkQoje1ZNn8vSx9pFH40 
